lang/it.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: it\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Utilizzo: vrms-rpm [opzione]"
+msgid "msg_help_option_ascii"    
+msgstr "    Mostra una ASCII-art di rms quando non esiste nessun pacchetto libero\n"
+       "    o quando i pacchetti non liberi sono piu' del 10%% del totale."
+msgid "msg_help_option_explain"
+msgstr "    Quando si elencano i pacchetti, mostra le licenze per giustificare\n"
+       "    la classificazione in  libero / non-libero."
+msgid "msg_help_option_help"
+msgstr "    Mostra questo messaggio di aiuto ed esce."
+msgid "msg_help_option_list"
+msgstr "    Oltre a mostrare un sommario del numero di pacchetti liberi & non,\n"
+       "    li stampa per nome. Il valore di default e' 'nonfree'."
+msgid "msg_help_option_version"
+msgstr "    Mostra informazioni sulla versione ed esce."
+msgid "msg_list_noargument"
+msgstr "%s: --list richiede un argomento"
+msgid "msg_list_badargument"
+msgstr "%s: l'argomento per l'opzione --list deve essere uno tra: 'none', 'free', 'nonfree' o 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: opzione sconosciuta '%s'"
+msgid "msg_total_free"
+msgstr "%d pacchetti liberi"
+msgid "msg_total_nonfree"
+msgstr "%d pacchetti non liberi"
+msgid "msg_rms_happy_ascii"
+msgstr "Solo pacchetti liberi - rms sarebbe orgoglioso!                           rms e' felice."
```

Comment:
Please remove some spaces to fit this line in 79 characters.

Reply:
Done!